nevergrad/optimization/base.py

```diff
@@ -314,6 +314,11 @@ def tell(self, candidate: p.Parameter, loss: tp.Loss) -> None:
         if isinstance(loss, (Real, float)):
             # using "float" along "Real" because mypy does not understand "Real" for now Issue #3186
             loss = float(loss)
+            # Non-sense values including NaNs should not be accepted.
+            # We do not use max-float as various later transformations could lead to greater values.
+            if not (loss < 5.0e20):
+                warnings.warn(f"Pathological value {loss} in tell.")
```

Comment:
More explicit message for the user:
```suggestion
                warnings.warn(f"Clipping very high value {loss} in tell (rescale the cost function?).")
```

Reply:
Thx!